_posts/2024-05-08-new-mailing-lists.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "New Mailing Lists for Sandstorm"
+author: Jacob Weisz
+authorUrl: https://github.com/ocdtrekkie
+---
+
+Several of us have long talked about how to get away from Google Groups and when to do
+so. Google has not gotten better at managing spam both in terms of junk registrations
+and emails, as well as recently flagging one of our core community member's posts as
+spam more than once, despite them being here for several years. There's also the
+fundamental risk Google shutters or substantially alters Groups, and of course, it
+always seems a bit odd to ask users of a privacy-centered project to sign up with Google. 
+
+We've decided to launch new mailing lists using [groups.io][1], a service founded by the
+original developer of Yahoo! Groups before it was acquired. It's a solid platform with a
+very generous free tier and feature set. The service has been running since 2014 and the
+founder started making mailing list software in 1998, so the likelihood of stability and
+longevity here seems pretty high. It has some pleasant features on their site, but like
+any good traditional mailing list software, can handle pretty much everything you can
+want entirely over email. 
+
+As we are focused on ethical practices and solid trust relationships, we are not opting
+anyone into the new mailing lists automatically. The individuals who subscribed to the
+Google Groups list did so in order to follow Sandstorm.io or Kenton, not necessarily us.
+Therefore, everyone does need to sign themselves up, and the lists will have a somewhat
+smaller reach, at least at first.
+
+There have often been questions about whether people could bring general questions or
+support requests to a "dev" list, so we've decided that instead of having a single list,
```

Comment:
I think you could get away with ending the sentence after "list" and ditching "so". Shorter sentences.

Reply:
Actually that's a bit off. Gimme a second...

Reply:
Nevermind I won't suggest any changes here.